client/dist/style.css

```diff
@@ -0,0 +1,160 @@
+.flex-container {
```

Comment:
not a priority right now and I don't think it should be an issue since I believe we're all using styled components but might want to refactor to styled components when you get the chance. Eric might have some class names actually so may want to double check those to make sure you don't have any overlaps.